paddle/operators/conv_shift_op.h

```diff
@@ -0,0 +1,135 @@
+/* Copyright (c) 2017 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#pragma once
+#include "paddle/framework/eigen.h"
+#include "paddle/framework/op_registry.h"
+
+namespace paddle {
+namespace operators {
+
+using Tensor = framework::Tensor;
```

Comment:
use `using` in the header file violate the google style,  see the detail in [Namespace](https://google.github.io/styleguide/cppguide.html#Namespaces)
> Do not use Namespace aliases at namespace scope in header files except in explicitly marked internal-only namespaces, because anything imported into a namespace in a header file becomes part of the public API exported by that file.

By the way,  I think
```c++
using framework::Tensor;
```
is enough here. 

Some old operators have a wrong guide, sorry for that.

Reply:
I just see this comment by chance.

The Google style says: 
> Do not use **Namespace aliases** at namespace scope in header files

I think:
```cpp
using Tensor = framework::Tensor
```
this usage does not violate the above Google style because it is just a type alias, not a namespace alias. But, I admit that using type alias in such a large scope is not a good thing, so you are right...